src/components/Accounts/Transactions/Transaction/index.js

```diff
@@ -29,7 +31,15 @@ export default class Transaction extends React.Component {
         onClick={transactionSelect}
       >
         <div className="col s10">
-          <img src={logo || require('assets/shopping-bag.svg')} alt={merchant} className="rounded circle" />
+          {logo ?
```

Comment:
Could you amend this code to follow the use brackets I use [here](https://github.com/robcalcroft/mondoweb/blob/master/src/components/Accounts/Transactions/index.js) in the ternary expressions thanks